.changeset/shiny-brooms-punch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ai-sdk/orchestration': minor
+---
+
+[Improvement] Add convenience function to create document grounding configuration.
```

Comment:
I in general feel using the term `document grounding` is a bit confusing considering we have a package called document grounding. Can we rename everywhere this term into orchestration grounding? Technically, this package has nothing to do with document grounding service, but calling only orchestration service.

Reply:
> has nothing to do with document grounding service

But that's not really true, semantically it's the same service. And it's actually important that users know this, otherwise they don't know that they need to use both things together for the setup to work.

It's just that we are using Document Grounding via Orchestration. And arguably, this is a technical detail users don't even (need to) know about. We could also just call DG directly in a separate call, before calling orchestration. It's rather an implementation detail I'd say.